slicer_configurations/PrusaSlicer/IamPerrine/Voron2_2.ini

```diff
@@ -0,0 +1,235 @@
+# generated by PrusaSlicer 2.1.0+win64 on 2020-01-30 at 23:10:55 UTC
+avoid_crossing_perimeters = 0
+bed_custom_model = C:\\Users\\jakep\\OneDrive\\3D Printer\\VORON\\voron2_350bed.stl
```

Comment:
This hard link to the file on your machine won't work anywhere else, I suggest to remove it.

Reply:
I'll make these changes. Do I need to push the update and make a new PR? Or will it update automatically? Sorry I'm a github noob still.